sisyphus/graph.py

```diff
@@ -230,16 +230,20 @@ def output(self):
 
     def add_target(self, target):
         """
-        :param OutputTarget target:
```

Comment:
add_target should handle all OutputTargets and not only path. It's currently also called by register_report and register_callback.